Description: Offline makefile
 Removes tests from Makefile that hit the Internet.
 .
 mdcheckr (1.1-1ubuntu1~trustyppa1) trusty; urgency=low
 .
   * Improvements to command-line interface
Author: Michael Billington <michael.billington@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-06-12

--- mdcheckr-1.1.orig/Makefile
+++ mdcheckr-1.1/Makefile
@@ -6,5 +6,6 @@ default: test
 
 test:
 	bash -n mdcheckr
-	(cd tests && ./test.sh)
+  # Tests hit the Internet, switching off for offline build
+	#(cd tests && ./test.sh)
 
